products/ubuntu.md

```diff
@@ -118,16 +117,39 @@ releases:
 
 During the lifetime of an Ubuntu release, Canonical provides security maintenance. Basic Security Maintenance covers binary packages that reside in the `main` and `restricted` components of the Ubuntu archive, typically for a period of 5 years from LTS release.
 
-Extended Security Maintenance (ESM) provides security updates on Ubuntu LTS releases for additional 5 years. It is available with the Ubuntu Pro subscription or a Free subscription. Please see the [Ubuntu Website](https://ubuntu.com/pro) for details.
+Packages in `universe` are expected to be community-supported on a best-effort basis during the LTS phase, but in practice, security-updates are only released to pro subscribtions.
+
+Extended Security Maintenance (ESM) provides security updates on Ubuntu LTS releases for additional 5 years. It is available with the [Ubuntu Pro](https://ubuntu.com/pro) subscription or a [Free subscription for personal use](https://ubuntu.com/blog/ubuntu-pro-beta-release)[^4].
+
+Ubuntu Pro offers security fixes for critical, high, and selected medium CVEs in the `main` and `universe` repositories. Ubuntu Pro (Infra-only)[^1] only guarantees security fixes for packages in the `main` repository.
+
+## Support Comparison
 
-The dates for active and security support are taken from [here](https://github.com/canonical-web-and-design/ubuntu.com/blob/master/static/js/src/chart-data.js) what is used for the graph rendering on the [Release Cycle Page](https://ubuntu.com/about/release-cycle).
+Plan | Ubuntu LTS | Ubuntu Pro (Infra-Only) [^1] | Ubuntu Pro
+-----|------------|------------------------------|-----------
+Main repository | 5 years | 10 years             | 10 years
+Restricted repository | 5 years | 10 years[^2]   |10 years [^2]
+Universe repository | Best Effort | Best Effort | 10 years 
+Phone/Ticket Support | No | Yes | Yes
+Kernel Live Patching | No  | Yes | Yes
+[Security Certifications and Hardening](https://ubuntu.com/security/certifications)[^3] | No | Yes | Yes
+[Ubuntu Landscape](https://ubuntu.com/landscape) | No | Yes | Yes
 
-Ubuntu Pro offers security fixes for critical, high, and selected medium CVEs in the `main` and `universe` repositories. Ubuntu Pro (Infra-only) - previously known as Ubuntu Advantage - only guarantees security fixes for packages in the `main` repository - other packages are on a best-effort basis.
+For package specific support details, the following commands are available:
+
+- `ubuntu-security-status` command (`ubuntu-support-status` on versions before `20.04`) for non-ESM.
+- [`pro security-status`](https://manpages.ubuntu.com/manpages/kinetic/en/man1/ubuntu-advantage.1.html) for ESM.
+
+## Release Cadence
```

Comment:
This section should be displayed in first. It would gently introduce the _LTS releases are in "General Support"..._ section.